streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -653,30 +653,6 @@ public void shouldThrowExceptionIfNotAtLeastOnceOrExactlyOnce() {
         assertThrows(ConfigException.class, () -> new StreamsConfig(props));
     }
 
-    @Test
```

Comment:
We should keep all these tests (cf. my other comment above).

We should also keep all text code below which set the config... (don't want to comment on each file, so just make this single comment here)